src/main.rs

```diff
@@ -25,6 +35,18 @@ use crate::setup_scripts::udc::UdcSetup;
 use crate::utils::banner::BANNER;
 use crate::utils::{save_to_json, JsonValueType};
 
+#[derive(Debug, Clone, Copy, ValueEnum)]
+enum BootstrapMode {
+    Core,
+    SetupL1,
+    SetupL2,
+    EthBridge,
+    Erc20Bridge,
+    Udc,
+    Argent,
+    Braavos,
+}
+
 #[derive(Parser, Debug)]
 #[command(version, about, long_about = None)]
 pub struct CliArgs {
```

Comment:
I guess a config file would make sense here, this would look hard to manage/update only through CLI (similar to the ChainConfig in madara)

Reply:
added